{
	"schema_version": "1.6.0",
	"id": "RHSA-2021:2704",
	"modified": "2024-12-16T10:03:27Z",
	"published": "2024-09-30T14:16:14Z",
	"related": [
		"CVE-2021-27918",
		"CVE-2021-31525",
		"CVE-2021-33196"
	],
	"summary": "Red Hat Security Advisory: Release of OpenShift Serverless Client kn 1.16.0",
	"severity": [
		{
			"type": "CVSS_V3",
			"score": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H"
		}
	],
	"affected": [
		{
			"package": {
				"ecosystem": "Red Hat:serverless:1.0::el8",
				"name": "openshift-serverless-clients",
				"purl": "pkg:rpm/redhat/openshift-serverless-clients"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:0.22.0-3.el8"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2021:2704.json"
			}
		}
	],
	"references": [
		{
			"type": "ADVISORY",
			"url": "https://access.redhat.com/errata/RHSA-2021:2704"
		},
		{
			"type": "ARTICLE",
			"url": "https://access.redhat.com/security/updates/classification/#moderate"
		},
		{
			"type": "ARTICLE",
			"url": "https://access.redhat.com/documentation/en-us/openshift_container_platform/4.6/html/serverless/index"
		},
		{
			"type": "ARTICLE",
			"url": "https://access.redhat.com/documentation/en-us/openshift_container_platform/4.7/html/serverless/index"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1937901"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1958341"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1965503"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1971449"
		},
		{
			"type": "ADVISORY",
			"url": "https://security.access.redhat.com/data/csaf/v2/advisories/2021/rhsa-2021_2704.json"
		},
		{
			"type": "REPORT",
			"url": "https://access.redhat.com/security/cve/CVE-2021-27918"
		},
		{
			"type": "ADVISORY",
			"url": "https://www.cve.org/CVERecord?id=CVE-2021-27918"
		},
		{
			"type": "ADVISORY",
			"url": "https://nvd.nist.gov/vuln/detail/CVE-2021-27918"
		},
		{
			"type": "ARTICLE",
			"url": "https://groups.google.com/g/golang-announce/c/MfiLYjG-RAw"
		},
		{
			"type": "REPORT",
			"url": "https://access.redhat.com/security/cve/CVE-2021-31525"
		},
		{
			"type": "ADVISORY",
			"url": "https://www.cve.org/CVERecord?id=CVE-2021-31525"
		},
		{
			"type": "ADVISORY",
			"url": "https://nvd.nist.gov/vuln/detail/CVE-2021-31525"
		},
		{
			"type": "ARTICLE",
			"url": "https://groups.google.com/g/golang-announce/c/cu9SP4eSXMc"
		},
		{
			"type": "REPORT",
			"url": "https://access.redhat.com/security/cve/CVE-2021-33196"
		},
		{
			"type": "ADVISORY",
			"url": "https://www.cve.org/CVERecord?id=CVE-2021-33196"
		},
		{
			"type": "ADVISORY",
			"url": "https://nvd.nist.gov/vuln/detail/CVE-2021-33196"
		},
		{
			"type": "ARTICLE",
			"url": "https://groups.google.com/g/golang-announce/c/RgCMkAEQjSI"
		}
	]
}
